components/MapMarkers.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Marker } from "react-native-maps";
+
+const MapMarkers = ({ markers, setRegionAndSelectLandmark }) => {
+  if (markers)
```

Comment:
is this if statement doing anything? the syntax without curly brackets is new to me.

Reply:
Good catch--fixed.